src/cmd/troubleshoot/component.go

```diff
@@ -5,10 +5,10 @@ import "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
 type component string
 
 const (
-	componentOneAgent    = "OneAgent"
-	componentCodeModules = "OneAgentCodeModules"
-	componentActiveGate  = "ActiveGate"
-	customImagePostfix   = " (custom image)"
+	oneAgent           component = "OneAgent"
```

Comment:
I would keep the "component" prefix in the const
```suggestion
	componentOneAgent           component = "OneAgent"
```